lib/prepArticles.js

```diff
@@ -49,7 +49,7 @@ function prepAll(store, callback) {
 		            	result.push(newArticle)
 		            }
 
-		            if (completedArticles === articleList.length){
+		            if (completedArticles >= articleList.length){
```

Comment:
Curious what this is for?

Reply:
Oh lol bad logic on my part, it's to account for Cole's encounter with the completed articles somehow exceeding the total amount, just in case.